ckb-bin/src/subcommand/run.rs

```diff
@@ -98,19 +104,39 @@ pub fn run(args: RunArgs, version: Version) -> Result<(), ExitCode> {
             move || Box::new(net_timer.clone()),
             Arc::clone(&network_state),
         ),
+        CKBProtocol::new(
+            "alt".to_string(),
+            NetworkProtocol::ALERT.into(),
+            &["1".to_string()][..],
+            move || Box::new(alert_relayer.clone()),
+            Arc::clone(&network_state),
+        ),
     ];
     let network_controller = NetworkService::new(Arc::clone(&network_state), protocols)
         .start(version, Some("NetworkService"))
         .expect("Start network service failed");
 
-    let rpc_server = RpcServer::new(
-        args.config.rpc,
-        network_controller,
-        shared,
-        synchronizer,
-        chain_controller,
-        block_assembler_controller,
-    );
+    let builder = ServiceBuilder::new(&args.config.rpc)
+        .enable_chain(shared.clone())
+        .enable_pool(shared.clone(), network_controller.clone())
+        .enable_miner(
+            shared.clone(),
+            network_controller.clone(),
+            chain_controller.clone(),
+            block_assembler_controller,
+        )
+        .enable_net(network_controller.clone())
+        .enable_stats(shared.clone(), synchronizer, Arc::clone(&alert_notifier))
+        .enable_experiment(shared.clone())
+        .enable_integration_test(
+            shared.clone(),
+            network_controller.clone(),
+            chain_controller.clone(),
+        )
+        .enable_alert(alert_verifier, alert_notifier, network_controller);
```

Comment:
Still has the problem I mentioned before. Is it by design that event RPC is disabled, we still need to relay alerts?

Reply:
Yes, the RPC is designed for administrators to send alerts; By default, every client needs to relayer alert.